collabora_fr.md

```diff
@@ -1,26 +1,31 @@
-# Installer Collabora avec Nextcloud
+# Installer Collabora avec Nextcloud via Docker
 
-**Note :** la marche à suivre detaillée et realisée ici à partir d’une instance Yunohost  sur Debian 8 (celle-ci n'a pas été testée suite a la migration sur la version 3 de Yunohost)  et celle-ci part du principe que les domaines/sous-domaines sont correctement configurés au niveau des DNS et de votre instance Yunohost (voir [DNS](/dns_fr) , [DNS et installation d’une application sur un sous-domaine](/dns_subdomains_fr) ,[Configurer les enregistrements DNS](/dns_config_fr) et [Nom de domaine en noho.st / nohost.me](/dns_nohost_me_fr)  )
+**Note :** la marche à suivre detaillée eet realisée ici à partir d’une instance Yunohost  sur Debian 8 (celle-ci n'a pas été testée suite à la migration sur la version 3 de Yunohost). Ces instructions ont pour pré-requis que les domaines/sous-domaines sont correctement configurés au niveau des DNS et de votre instance Yunohost (voir [DNS](/dns_fr) , [DNS et installation d’une application sur un sous-domaine](/dns_subdomains_fr) ,[Configurer les enregistrements DNS](/dns_config_fr) et [Nom de domaine en noho.st / nohost.me](/dns_nohost_me_fr) )
```

Comment:
"eet"; 
double-espace; 
"sur" : je préfère "vers" ; 
add "/ynh.fr" as it's the new dyndns domain.